src/components/screens/signMultiSignTransaction/importData/importData.js

```diff
@@ -23,23 +23,31 @@ const ImportData = ({ t, nextStep }) => {
     }
   };
 
+  const resetState = () => {
+    setTransaction(undefined);
+    setError(undefined);
+  };
+
   const validateAndSetTransaction = (input) => {
     try {
       const parsedInput = JSON.parse(input);
       setTransaction(parsedInput);
     } catch (e) {
+      setTransaction(input);
```

Comment:
I think it might be better to set this to `setTransaction(undefined)` instead to avoid this behaviour where the clipboard or the textarea component seems to have added some escapes